includes/data-stores/class-wc-admin-reports-orders-data-store.php

```diff
@@ -289,8 +291,24 @@ protected function include_extended_info( &$orders_data, $query_args ) {
 			);
 		}
 
+		foreach ( $coupons as $coupon ) {
+			if ( ! isset( $mapped_data[ $coupon['order_id'] ] ) ) {
+				$mapped_data[ $product['order_id'] ]['coupons'] = array();
+			}
+
+			$mapped_data[ $coupon['order_id'] ]['coupons'][] = array(
+				'id'   => $coupon['coupon_id'],
+				'code' => $coupon['coupon_code'],
```

Comment:
I think this should be wrapper around `wc_format_coupon_code()`:

```suggestion
				'code' => wc_format_coupon_code( $coupon['coupon_code'] ),
```


I have an uppercase and a lowercase code, and they are both displayed lowercase everywhere except here:
![image](https://user-images.githubusercontent.com/3616980/51335267-ef55ed00-1a81-11e9-8e97-345da4749e91.png)

It should be:
![image](https://user-images.githubusercontent.com/3616980/51335304-0c8abb80-1a82-11e9-8b8a-34f956742824.png)